src/components/form/Obs.jsx

```diff
@@ -13,9 +13,10 @@ import withFormContext from './withFormContext';
 const Obs = (props) => {
 
   const conceptAnswerDisplay = (value, conceptAnswers) => {
+
     if (value) {
       const matchingAnswer = conceptAnswers.find(ans => ans.uuid === value);
-      return matchingAnswer ? matchingAnswer.name : null;
+      return matchingAnswer ? matchingAnswer.name.display : null;
```

Comment:
@cioan From the way the `.name` is being used multiple places in the <Obs />, it seems `.name` is a string, however I mostly encounter it as an object. seems @rhenshaw56 encountered it as an object too, hence his changes.
![screen shot 2018-10-03 at 12 16 12 pm](https://user-images.githubusercontent.com/26272732/46407333-8632af80-c706-11e8-8ce4-8797bb0651d8.png)

Any ideas on this?

Reply:
In any other openmrs rest responses, i think name is always a string, but maybe here it means ConceptName object? what is the web service request you send?

Reply:
yeah @cioan I think it's the `conceptName` object because for concepts basically, the name field is always an object and from the way the variable was named conceptAnswers, I figured it always had to be a concept, unless there are other cases

Reply:
ok. you might want to change this line: 
> return matchingAnswer ? matchingAnswer.name.display : null;
to
> return matchingAnswer ? matchingAnswer.name.name : null;

Reply:
hi @cioan I've applied the changes to this PR https://github.com/openmrs/openmrs-react-components/pull/40